docs/Apps/Metabolomic Data/Polly-PeakML.md

```diff
@@ -0,0 +1,275 @@
+
+Polly-PeakML is a novel machine-learning tool that enables the curation of peak groups into real-signal and noise and provides additional information on correlating features and variation across cohorts, in a matter of minutes. 
+
+## Scope
+
+* Make use of [El-MAVEN](https://resources.elucidata.io/elmaven) for automated peak-picking and displaying curated results.
+* Raw data to curated peak-groups in minutes.
+* Process high-resolution and low-resolution data of up to 100 samples.
+
+![Polly-PeakML workflow schematic](../../img/Polly-PeakML/Workflow.png) <center>**Figure 1.** Polly-PeakML workflow</center>
+
+# Getting Started
+
+## User input
+
+Polly-PeakML is capable of processing the file-format supported by El-MAVEN.
+
+PeakML requires following two types of files: 
+
+**Sample Files**
+
+Followings sample file formats are supported -
+
+* .mzXml / .mzxml
+* .mzMl /  .mzml
+* .emDB / .mzroll
+* .mzPeaks
+
+**Cohort Information File**
+
+Following cohort file format is supported -
+
+* .CSV / .csv
+
+## Steps involved in the data processing 
+
+**Load samples and cohort file**
+
+Users can load the sample by going to Load Samples|Workspaces|Peaks option in the Sample space menu. Then navigate to the folder containing the sample data and select all .mzXML or .mzML files. A loading bar displays the progress at the bottom. When the samples have loaded, you should see a sample panel on the left side.
+
+Likewise, you can load the cohort information by choosing the *upload* option in the sample space menu. 
+
+The uploaded information would be rendered on El-MAVEN as shown in figure 2.
+
+![Sample upload](../../img/Polly-PeakML/sampleUpload.png) <center>**Figure 2.** Upload samples and cohort information</center>
+
+**Adjust global parameters**
+
+Change global settings for El-MAVEN by clicking the *settings* icon given on the menu bar of the appliication. 
+
+![Settings Dialog](../../img/Polly-PeakML/settingsDialog.png) <center>**Figure 3.** Global Settings Dialog</center>
+
+**Perform Alignment**
+
+Alignment shifts the peak RTs in every sample to correct for this drift and brings the peaks closer to median RT of the group. *Alignment visualization* can be used to judge the extent of deviation from median RT. 
+
+Users can perform alignment by going to the option *Alignment* given on the menu bar of application.
+
+![Alignment Dialog](../../img/Polly-PeakML/alignment.png) <center>**Figure 4.** Alignment Dialog</center>
+
+**Peak Detection and Curation**
+
+* Select Peak Detection flow - Targeted or Untargeted pipeline.
```

Comment:
Replace untargeted and targeted with automated feature detection and compound database search and make them italic